systemd/csp-billing-adapter.service

```diff
@@ -0,0 +1,12 @@
+[Unit]
+Description=CSP usage reporting
+Wants=billing-data-service.service
```

Comment:
Guess this is product side?

Reply:
Yes, the SUMa team created a separate service for the API.